README.md

```diff
@@ -19,7 +19,7 @@ installing the `ocne-catalog` chart.
 | Cluster API Kubeadm Control Plane Controller        | control-plane-capi       | 1.7.1    |
 | Cluster API Kubeadm Bootstrap Controller            | bootstrap-capi           | 1.7.1    |
 | Cluster API for Oracle Cloud Infrastructure         | oci-capi                 | 0.15.0   |
-| Cert Manager                                        | cert-manager             | 1.14.5   |
```

Comment:
We need to retain the old version in this list as it is still supported.

Reply:
fixed